Makefile

```diff
@@ -13,13 +13,17 @@ REPO := "https://github.com/metal-toolbox/flasher.git"
 .DEFAULT_GOAL := help
 
 ## lint
-lint:
+lint: gen-mock
 	golangci-lint run --config .golangci.yml
 
 ## Go test
-test:
+test: lint
 	CGO_ENABLED=0 go test -timeout 1m -v -covermode=atomic ./...
 
+## Generate mocks
+gen-mock:
+	mockgen -source internal/model/model.go -destination internal/fixtures/mock.go -package fixtures
```

Comment:
As a convenience, you might want to add an install command to make sure the user has gomock installed:
```
go install github.com/golang/mock/mockgen@v1.6.0
```

And a more scalable way to manage gomocks is with `go generate`. You just have to add the `go:generate` annotations in your code. Then `go generate ./...` will find those annotations and generate all the necessary code. Otherwise, every time you create a new mock, you'll end up editing the Makefile.
More info here: https://gist.github.com/thiagozs/4276432d12c2e5b152ea15b3f8b0012e#usinggomockwithgogenerate

Then this make target should look like this, and you'll update it a lot less frequently:
```
gen-mock:
    go install github.com/golang/mock/mockgen@v1.6.0
    go generate ./...
```

Reply:
TIL, thanks for the pointers!